packages/@aws-cdk/aws-cloudformation/lib/pipeline-actions.ts

```diff
@@ -366,8 +364,17 @@ export enum CloudFormationCapabilities {
    * names.
    *
    * `CloudFormationCapabilities.NamedIAM` implies `CloudFormationCapabilities.IAM`; you don't have to pass both.
+   * @link https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-iam-template.html#using-iam-capabilities
+   */
+  NamedIAM = 'CAPABILITY_NAMED_IAM',
+
+  /**
+   * No IAM Capabilities
+   *
+   * Pass this capability if you wish to block the creation IAM resources.
+   * @link https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-iam-template.html#using-iam-capabilities
    */
-  NamedIAM = 'CAPABILITY_NAMED_IAM'
+  None = '',
```

Comment:
I would actually make `None` the first enum value (I would sort them in the order of increasing capabilities).